scripts/install/readarr.sh

```diff
@@ -0,0 +1,143 @@
+    case "$(_os_arch)" in
+        "amd64") dlurl="${urlbase}&arch=x64" ;;
+        "armhf") dlurl="${urlbase}&arch=arm" ;;
+        "arm64") dlurl="${urlbase}&arch=arm64" ;;
+        *)
+            echo_error "Arch not supported"
+            exit 1
+            ;;
+    esac
+
+    if ! curl "$dlurl" -L -o "/tmp/$app_name.tar.gz" >> "$log" 2>&1; then
+        echo_error "Download failed, exiting"
+        exit 1
+    fi
+    echo_progress_done "Archive downloaded"
+
+    echo_progress_start "Extracting archive"
+    tar xfv "/tmp/$app_name.tar.gz" --directory /opt/ >> "$log" 2>&1 || {
+        echo_error "Failed to extract"
+        exit 1
+    }
+    rm -rf "/tmp/$app_name.tar.gz"
+    chown -R "${user}": "$app_dir"
+    echo_progress_done "Archive extracted"
+
+    echo_progress_start "Installing Systemd service"
+    cat > "/etc/systemd/system/$app_servicefile" << EOF
+[Unit]
+Description=${app_name^} Daemon
+After=syslog.target network.target
+
+[Service]
+# Change the user and group variables here.
+User=${user}
+Group=${app_group}
+
+Type=simple
+
+# Change the path to ${app_name^} here if it is in a different location for you.
+ExecStart=$app_dir/$app_binary -nobrowser -data=$app_configdir
+TimeoutStopSec=20
+KillMode=process
+Restart=on-failure
+
+# These lines optionally isolate (sandbox) ${app_name^} from the rest of the system.
+# Make sure to add any paths it might use to the list below (space-separated).
+#ReadWritePaths=$app_dir /path/to/media/folder
+#ProtectSystem=strict
+#PrivateDevices=true
+#ProtectHome=true
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+    systemctl -q daemon-reload
+    systemctl enable --now -q "$app_name"
+    sleep 1
+    echo_progress_done "${app_name^} service installed and enabled"
+
+    echo_progress_start "${app_name^} is installing an internal upgrade..."
+    if ! timeout 30 bash -c -- "while ! curl -sIL http://127.0.0.1:$app_port >> \"$log\" 2>&1; do sleep 2; done"; then
+        echo_error "The ${app_name^} web server has taken longer than 30 seconds to start."
+        exit 1
+    fi
+    echo_progress_done "Internal upgrade finished"
+
+}
+
+_nginx_readarr() {
+    if [[ -f /install/.nginx.lock ]]; then
+        echo_progress_start "Configuring nginx"
+        sleep 10
+        bash /usr/local/bin/swizzin/nginx/"$app_name".sh
+        systemctl reload nginx
+        echo_progress_done "Nginx configured"
+    else
+        echo_info "$app_name will run on port $app_port"
+    fi
+}
+
+_calibre_cs_readarr() {
+    if [[ -f /install/.calibre.lock ]]; then
+        if ! systemctl -q is-active $app_servicename; then
+            if ask "Enable Calibre's Content Server for Readarr integration?" Y; then
+                systemctl enable --now -q $app_servicename
+            else
+                return 0
+            fi
+        fi
+        # We should set up the library here actually
+        # We know the location, ports, the user and password, so I see no reason not to
+    fi
```

Comment:
This should just offer to install `calibrecs` and `calibre`.

Do we want to look into setting those libraries up using the Readarr API?

Reply:
This was from your previous commits to this PR.

your call and I'd suggest not; let the user setup and determine if they wanted to use calibre

Reply:
```suggestion
    if [[ -f /install/.calibre.lock ]]; then
        if ! systemctl -q is-active $app_servicename; then
            if ask "Enable Calibre's Content Server for Readarr integration?" Y; then
                systemctl enable --now -q $app_servicename
            else
                return 0
            fi
        fi
    else
        echo_info "It is highly recommended that you install Calibre and its Content Server, you can do that with \`box install calibre calibrecs\`
    fi
```

Reply:
This change would require us to merge #401 before this, FYI